src/transformation/utils/annotations/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from "./collect";
+export * from "./utils";
```

Comment:
Might as well just put the contents of collect and/or utils in here? This seems a little useless.